Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mongo
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN chmod +x ./entrypoint.sh
+RUN apt-get update && apt-get install -y ruby
```

Comment:
Nitpick: might be an idea to put this earlier on in the dockerfile, so we can take advantage of the cache for this step. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds has more context

Reply:
thanks for the link - done!